Makefile

```diff
@@ -127,3 +127,7 @@ GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
 rm -rf $$TMP_DIR ;\
 }
 endef
+
+helm:
+	cp -rf deployment/crds charts
+	helm package charts
```

Comment:
The make target is adding a dependency on the external `helm` tool. We either download that within the make file. Let's handled it in a separate PR.